indexer/feature_data.hpp

```diff
@@ -66,10 +66,16 @@ namespace feature
     }
 
     /// Accumulation function.
-    inline void operator() (uint32_t type)
+    void Add(uint32_t type)
```

Comment:
Why do you need two functions with exact behavior? Please, leave just one of them.